app/javascript/selectors/detailSelector.js

```diff
@@ -181,3 +182,8 @@ export const formattedPhoneNumber = state => {
 export const phoneExtension = state => {
   return safeGet(state, 'fetchDetails.details.records.user.phone_extension_number') || ''
 }
+
+export const userNotes = state => {
+  const notes = safeGet(state, 'fetchDetails.details.records.user.notes') || ''
```

Comment:
can the || '' be changed to
```suggestion
  const notes = safeGet(state, 'fetchDetails.details.records.user.notes', '')
```
?

Reply:
@kelfink  That works when notes are undefined, But when notes are null it still gives me the null while testing the selector, So that's why i used that || '' to return '' if notes are null.

Reply:
cool, good enough of a reason.